tools/shared/ccffinfo.c

```diff
@@ -54,7 +54,11 @@ static int prevchildnest = -1;
 static int prevlineno = 0;
 static bool anymessages;
 
+#ifdef FLANG_VENDOR
+#define BUILD_VENDOR FLANG_VENDOR
```

Comment:
IMHO this belongs in the `version` structure as well.

Reply:
I agree, but this isn't a refactor patch.